node-it/src/test/scala/com/wavesplatform/it/sync/smartcontract/InvokeScriptTransactionStateChangesSuite.scala

```diff
@@ -234,6 +236,63 @@ class InvokeScriptTransactionStateChangesSuite extends BaseTransactionSuite with
     )
   }
 
+  test("state changes order") {
+    val script = ScriptCompiler
+      .compile(
+        """
+      |{-# STDLIB_VERSION 4 #-}
+      |{-# CONTENT_TYPE DAPP #-}
+      |{-# SCRIPT_TYPE ACCOUNT #-}
+
+      |@Callable(i)
+      |func orderTest() = {
+      |    [ScriptTransfer(i.caller, 1, unit),
+      |    IntegerEntry("a", 1),
+      |    StringEntry("b", "a"),
+      |    BinaryEntry("c", base58'a'),
+      |    BooleanEntry("y", true),
+      |    ScriptTransfer(i.caller, 2, unit),
+      |    StringEntry("b", "c")]
```

Comment:
please add the other types of actions: DeleteEntry, Issue, Reissue, Burn, SponsorFee